modules/phase_field/src/materials/ComputePolycrystalElasticityTensor.C

```diff
@@ -48,9 +48,6 @@ ComputePolycrystalElasticityTensor::computeQpElasticityTensor()
   const std::vector<std::pair<unsigned int, unsigned int> > & active_ops = _grain_tracker.getElementalValues(_current_elem->id());
```

Comment:
It's a speed versus space trade-off. I'm just not crazy about building both data structures. I suppose that I could build one or the other on demand so they usually aren't both built. On the other hand I doubt that looping over the vector is going to be a bottleneck. Vector looping is one of the fastest operations on a CPU.